cli/cmd/cluster_create.go

```diff
@@ -60,9 +62,41 @@ func (r *runners) createCluster(_ *cobra.Command, args []string) error {
 		return fmt.Errorf("%s\n\nSupported Kubernetes distributions and versions are:\n%s", errors.New(strings.Join(ve.Errors, ",")), supportedDistributions(ve.SupportedDistributions))
 	}
 
+	// if the wait flag was provided, we poll the api until the cluster is ready, or a timeout
+	if r.args.createClusterWait {
+		return r.waitForCluster(cl.ID)
+	}
+
 	return print.Cluster(r.outputFormat, r.w, cl)
 }
 
+func (r *runners) waitForCluster(id string) error {
+	maxWaitDuration := time.Minute * 5
```

Comment:
Can we make the `maxWaitDuration` configurable?
Or make it `r.args.createClusterMaxWaitDuration` with a default of `0` meaning no wait.